package.nls.json

```diff
@@ -37,6 +37,7 @@
     "AWS.command.deleteLambda.confirm": "Are you sure you want to delete lambda function '{0}'?",
     "AWS.command.deleteLambda.error": "There was an error deleting lambda function '{0}'",
     "AWS.command.deploySamApplication": "Deploy SAM Application",
+    "AWS.command.getToolkitInfo": "Get AWS Toolkit Info",
```

Comment:
I wonder if we should change this to something like "About the AWS Toolkit..."

Reply:
Changed to "About AWS Toolkit"